README.md

```diff
@@ -26,7 +26,94 @@ Estimating the geographical range of a species from sparse observations is a cha
 ```
 
 #### Data Download and Preparation
-Instructions for downloading the data in `data/README.md`.
+# Instructions for Data Preparation
```

Comment:
There is a problem with the headers here and in general. If you see the README on the GUI, it is hard to follow the data section and below as well (You can see it by opening your branch on the web view). Make sure the headers' style are consistent and there aren't duplicate titles (e.g. "Data Download and Preparation" and "Instructions for Data Preparation").

Reply:
pushed new changes